css-nav-1/Overview.bs

```diff
@@ -1348,56 +1381,95 @@ Note: the 'spatial-navigation-contain' property is <a>at-risk</a>.
+or whether the document should be scrolled in that direction.
+By default, this is automatically determined,
+but this property allows the author decide between focusing or scrolling.
+
+The precise behavior is defined in [[#nav]],
+but a high level description of the effect of each value is provided below.
+
+When spatial navigation is triggered,
+the behavior depends on the value of the 'spatial-navigation-action' on the currently focused element
+if that element is a [=scroll container=],
+or of its nearest [=scroll container=] ancestor if it isn't.
 
 <dl dfn-for=spatial-navigation-action dfn-type=value>
     <dt><dfn>auto</dfn>
-    <dd>If the element is a <a>scroll container</a>,
-        then directional input moves the focus to the candidates in the optimal viewing region.
-        When there isn't any visible candidate, then the container scrolls.
-        
-        Otherwise, the focus moves to the candidates in the viewport.
+    <dd>
+        If there are visible focusable elements within the
+        [=scroll container=] in the direction requested,
+        the closest one becomes focused.
+        Otherwise, the [=scroll container=] is scrolled in the direction requested.
 
     <dt><dfn>focus</dfn>
-    <dd>The directional input moves the focus to all candidates inside a scroll container. The container <a>cannot be scrolled manually</a>.
+    <dd>
+        The focus is moved to the nearest focusable element within the [=scroll container=],
+        regardless of whether it is visible.
+        If there are none,
+        the scroll container is <em>not</em> scrolled,
+        and the search continues up the tree instead.
 
-    NOTE: If the 'focus' value is given to 'spatial-navigation-action', 
-    <a event>navnotarget</a> event occurs when there isn’t any visible candidate in the given direction within the viewport of 
-    the <a>spatial navigation container</a> even through the container can be scrolled more.
+        Note: The [=scroll container=] may be scrolled as a side effect of focusing
+        an element which was previously not in view,
+        but it will not be scrolled directly.
 
-    <dt><dfn>scroll</dfn>
-    <dd>If the element is a <a>scroll container</a>,
-        then directional input only works for manually scrolling the container.
+        Note: If the ''spatial-navigation-action/focus'' value is given to 'spatial-navigation-action',
+        <a event>navnotarget</a> event occurs when there isn’t any visible candidate in the given direction within the viewport of
+        the <a>spatial navigation container</a> even if the container can be scrolled more.
 
-    Note: The 'scroll' value is <a>at-risk</a>. 
+    <dt><dfn>scroll</dfn>
+    <dd>
+        If the currently focused element is not itself a [=scroll container=],
+        this value on an ancestor [=scroll container=] has the same effect as ''spatial-navigation-action/auto''.
```

Comment:
This can be misinterpreted that the ancestor in this case always behaves with 'auto' value.
Adding more detail about this is only for the default case would be helpful.

Reply:
Do you have a better suggestion? Otherwise, there's all the details in section 8.2, so I think it is OK.

Reply:
You're right. I didn't connect to that section. I agree with you.